frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -294,3 +294,24 @@ Feature: Topology chart area
               And user drag the connector from the deployment workload
               And user drops the connector on the enabled bindable resource
              Then user will see service binding connection
+
+
+        @regression @odc-6361
+        Scenario: Search with label: T-06-TC26
+            Given user has created a deployment workload "nodejs-1"
+              And user has created a deployment workload named "nodejs-2"
+              And user is at Topology page chart view
+             When user selects "Label" option in filter menu
+              And user searches for label "app.kubernetes.io/component=nodejs-1"
+             Then user can see the workload "nodejs-1" visible
+
+
+        @regression @odc-6361 @manual
+        Scenario: Check last selected node in topology per project per session: T-06-TC26
+            Given user has created a deployment workload "nodejs-ex-1"
+              And user has created a deployment-config workload named "nodejs-ex-2"
+              And user is at Topology page chart view
+             When user clicks on workload "nodejs-ex-2" to open sidebar
+              And user opens the console in a new browser window
+              And user log in to the console
+             Then user will see the the workload "nodejs-ex-2" selected with sidebar open
```

Comment:
We remember the selected node per session. So, this scenario will not work if you open the console again in the new browser. 
The scenario should be something like this.
```suggestion
        @regression @odc-6361 @manual
        Scenario: Check last selected node in topology per project per session: T-06-TC26
            Given user has created a deployment workload "nodejs-ex-1"
              And user has created a deployment-config workload named "nodejs-ex-2"
              And user is at Topology page chart view
             When user clicks on workload "nodejs-ex-2" to open sidebar
              And user opens the details page "nodejs-ex-2" by clicking on the title 
              And user navigate back to Topology page 
             Then user will see the the workload "nodejs-ex-2" selected with sidebar open
```

Reply:
Updated